packages/design-system-react/scripts/create-component/create-component.ts

```diff
@@ -0,0 +1,109 @@
+import { promises as fs } from 'fs';
```

Comment:
we probably want to add a test for this

Reply:
Added! https://github.com/MetaMask/metamask-design-system/pull/139/files#diff-d57bf5b2bc5b03c79c00c81b80dd2089732493986d2ae7d4907fcc3e24d40099